catalogue/administration.yml

```
@@ -0,0 +1,36 @@
+category: Administration
+choices:
+  - name: nmap
+    Provider: nmap
+    description: |
+      utility for network discovery and security auditing.
+    actions: rpm:nmap
+
+  - name: virt-manager
+    provider: virt-manager
+    description: |
+      Desktop tool for managing virtual machines via libvirt.
+    actions: rpm:virt-manager
+
+  - name: Tailscale
+    provider: Tailscale
+    description: |
```

empty description. By the way, you don't need to always add `|`, adding `|` means you can add the text with a new line.
```suggestion
    description: A description meow
```